### PR TITLE
Fix RoutePrefix_Empty response issue

### PR DESCRIPTION
**Pull request by WeihanLi**
Additional to #2214 

Add test case for RoutePrefix_Empty

### CODE REVIEW

**Comment by WeihanLi** [comment]
Hi @domaindrivendev , I add a new test project for the RoutePrefix is the empty case, please let me if I'm wrong way, thanks

File: src/Swashbuckle.AspNetCore.SwaggerUI/SwaggerUIMiddleware.cs [hunk 1]
```diff
@@ -68,7 +66,7 @@ public async Task Invoke(HttpContext httpContext)
                 return;
             }
 
-            if (httpMethod == "GET" && $"/{_options.RoutePrefix}/index.html".Equals(path, StringComparison.OrdinalIgnoreCase))
+            if (httpMethod == "GET" && $"{$"/{_options.RoutePrefix}".TrimEnd('/')}/index.html".Equals(path, StringComparison.OrdinalIgnoreCase))
```
[the remark this answers is not in the document]
Comment (WeihanLi) [reply src/Swashbuckle.AspNetCore.SwaggerUI/SwaggerUIMiddleware.cs:69]: Yeah, reasonable, thanks for your valuable feedback
